### PR TITLE
Symfony 3.0 support

### PR DESCRIPTION
**Pull request by idildin**
#212

### CODE REVIEW

File: Form/Type/TranslationsFormsType.php [hunk 1]
```diff
@@ -62,14 +64,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     public function buildView(FormView $view, FormInterface $form, array $options)
     {
         $view->vars['default_locale'] = $this->localeProvider->getDefaultLocale();
-        $view->vars['required_locales'] = $options['required_locales'];
-    }   
+        $view->vars['required_locales'] = $this->localeProvider->getRequiredLocales();
```
Comment (webda2l) [inline Form/Type/TranslationsFormsType.php]: This change the logic if someone want override required locales for specific form

File: Form/Type/TranslationsType.php [hunk 2]
```diff
@@ -50,15 +51,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
      */
     public function buildView(FormView $view, FormInterface $form, array $options)
     {
-        $view->vars['default_locale'] = $options['default_locale'];
-        $view->vars['required_locales'] = $options['required_locales'];
+        $view->vars['default_locale'] = $this->localeProvider->getDefaultLocale();
+        $view->vars['required_locales'] = $this->localeProvider->getRequiredLocales();
```
Comment (webda2l) [inline Form/Type/TranslationsType.php]: Same here, this can be override for specific form

**Comment by tobias-93** [comment]
ping @idildin @webda2l

**Comment by webda2l** [comment]
Ok, thanks @idildin 

@tobias-93 Give a try to ````"a2lix/translation-form-bundle": "@dev"```` and I will tag a new release a bit later if all is right

**Comment by tobias-93** [comment]
@webda2l Everything works fine without deprecation warnings, please tag :)
@idildin :+1:

**Comment by webda2l** [comment]
Ok done as https://github.com/a2lix/TranslationFormBundle/releases/tag/2.1

**Pull request merged**